FAQ.md

```diff
@@ -288,3 +288,7 @@ By default, APISIX only listens on port 9080 when handling HTTP requests. If you
     ```
 
 2. Reload or restart APISIX
+
+## How could I only enable passive health check?
```

Comment:
@tzssangglass sorry for the trouble, I update the title of issue #2748 

the new title is: "It is unreasonable to only enable passive checking"

for more information, please take a look at: https://github.com/apache/apisix/issues/2746#issuecomment-727174592

Reply:
ok, I close this PR